.github/workflows/ci.yml

```
@@ -273,8 +252,8 @@ jobs:
         if: "!cancelled()"
         run: docker compose logs iperf3
 
-  compare-results:
-    if: github.event_name == 'pull_request'
+  upload_bencher:
```

nit: naming convention

```suggestion
  upload-bencher:
```